src/Sylius/Bundle/CoreBundle/Command/CheckRequirementsCommand.php

```diff
@@ -14,11 +14,25 @@
 namespace Sylius\Bundle\CoreBundle\Command;
 
 use RuntimeException;
+use Sylius\Bundle\CoreBundle\Installer\Checker\RequirementsCheckerInterface;
+use Sylius\Bundle\CoreBundle\Installer\Checker\SyliusRequirementsChecker;
+use Sylius\Bundle\CoreBundle\Installer\Requirement\Requirement;
+use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
-final class CheckRequirementsCommand extends AbstractInstallCommand
+final class CheckRequirementsCommand extends Command
 {
+    /** @var RequirementsCheckerInterface */
+    private $requirementsChecker;
+
+    public function __construct(RequirementsCheckerInterface $requirementsChecker)
```

Comment:
🚨 BC Break 

Won't it be possible to leave `AbstractInstallCommand` as a parent class and allow `RequirementsCheckerInterface` to be null? In such a case, we can trigger deprecation notice and assign one from the container.